rasa/nlu/data_augmentation/paraphraser.py

```diff
@@ -0,0 +1,554 @@
+import collections
```

Comment:
I think we should change the name of this file. It's not really `paraphraser`. Could be `augmenter` or something similar.